src/commands.ts

```diff
@@ -157,9 +157,9 @@ export class Commands<C extends Context> {
                     language_code: language === "default"
                         ? undefined
                         : language,
-                    commands: commands.map((command) =>
-                        command.toObject(language)
-                    ),
+                    commands: commands
+                        .filter((command) => typeof command.name === "string")
+                        .map((command) => command.toObject(language)),
```

Comment:
I actually think it's better if we move this kind of thing to the actual `setCommands` method, so users can still use `toArgs` and `toSingleScopeArgs` however they want. WDYT?

Reply:
Yeah, makes more sense

Reply:
output looks like:
```ts
[
  {
    name: "bread",
    language: "default",
    prefix: "/",
    scopes: [ { type: "default" } ],
    description: "_"
  },
  {
    name: "pan",
    language: "es",
    prefix: "/",
    scopes: [ { type: "default" } ],
    description: "_"
  },
// etc
```

Reply:
Hm... yeah, that makes sense. Thanks!